chains/irisnet/handlerIrisnet.go

```diff
@@ -971,7 +1031,7 @@ func getPrivateKey() ed25519.PrivKeyEd25519 {
 
 // ---------------------- COMMON UTILITIES ---------------------------------
 
-
+//Creates Tendermint Handler between Marlin Relay and TM Core
```

Comment:
creates between relay and TM core? no really... it just creates a handler object